server/src/test/java/org/apache/druid/metadata/IndexerSQLMetadataStorageCoordinatorTest.java

```diff
@@ -138,16 +138,19 @@ public void testCommitAppendSegments()
     final String v1 = "2023-01-01";
     final String v2 = "2023-01-02";
     final String v3 = "2023-01-03";
+    final String alreadyUpgradedVersion = "2023-02-01";
     final String lockVersion = "2024-01-01";
 
-    final String replaceTaskId = "replaceTask1";
+    final String taskAllocatorId = "appendTask";
+    final String replaceTaskId = "replaceTask";
```

Comment:
Task ID seems to have changed. Just curious about why this was needed?

Reply:
I may have done it to eliminate the 1 at the end
Let me revert it

Reply:
reverted